clusters/build-clusters/01_cluster/_install/install_cluster.sh

```diff
@@ -20,22 +12,27 @@ readonly SCRIPT
 
 if [[ "$#" -lt  1 ]]; then
   >&2 echo "[FATAL] Illegal number of parameters"
-  >&2 echo "[INFO] usage: $SCRIPT <release> [path_to_install_config]"
-  >&2 echo "[INFO] e.g., $SCRIPT registry.svc.ci.openshift.org/ocp/release:4.2.0-0.nightly-2019-10-08-232417"
+  >&2 echo "[INFO] usage: $SCRIPT <ocp_version> [path_to_install_config]"
+  >&2 echo "[INFO] e.g., $SCRIPT 4.3.0"
   exit 1
 fi
 
-RELEASE_IMAGE="$1"
-readonly RELEASE_IMAGE
-
-OC_CLI="/usr/bin/oc"
-readonly OC_CLI
+OCP_VERSION="$1"
+readonly OCP_VERSION
 
 INSTALL_FOLDER="${HOME}/install_openshift/$(date '+%Y%m%d_%H%M%S')"
 readonly INSTALL_FOLDER
 mkdir -pv "${INSTALL_FOLDER}"
 
-"$OC_CLI" adm release extract --tools "${RELEASE_IMAGE}" --to="${INSTALL_FOLDER}"
+unameOut="$(uname -s)"
+readonly unameOut
+case "${unameOut}" in
+    Linux*)     machine=linux;;
+    Darwin*)    machine=mac;;
+    *)          >&2 echo "[FATAL] Unknow OS" and exit 1;;
+esac
+
+curl -o "${INSTALL_FOLDER}/openshift-install-${machine}-${OCP_VERSION}.tar.gz" https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.3.0/openshift-install-${machine}-${OCP_VERSION}.tar.gz
```

Comment:
The `mirror.openshift.com` URL needs to have the version parameter, too -- right now you hardcode 4.3

Reply:
Done.